INSTALL

```diff
@@ -143,6 +143,45 @@ and then install with:
 
     make install
 
+By default, rpm installs a series of default platforms based on the CPU
+architecture names in subdirectories called
+
+    /usr/lib/platform/<arch>-linux
+
+This is enough for regular Linux distributions, except Yocto. Yocto has
+so called Board Support Packages (BSPs) that cover more than the CPU
+architectures. These BSPs cover specific single board computer systems.
+Examples of such BSP names:
```

Comment:
Can you reword this to not require knowing what Yocto is? That is, if Yocto disappeared tomorrow, we wouldn't have a good reference for the need for this feature.

While we want to serve and support distributions, we try to ensure our features aren't "distro-dependant", so if the documentation on this can be made compelling without singling out or requiring to know Yocto, that would significantly help.